jax/experimental/compilation_cache/compilation_cache.py

```diff
@@ -14,7 +14,44 @@
 
 import hashlib
 import jax
+from jax.experimental.compilation_cache.file_system_cache import FileSystemCache
+from jax.lib import xla_client
+from typing import Optional
+import warnings
 
+cache = None
+
+def initialize_cache(path):
+    """Creates a global cache object. Should only be called once per process."""
+    global cache
+    cache = FileSystemCache(path)
+
+def get_executable(xla_computation, compile_options) -> Optional[xla_client.Executable]:
+    """Returns the cached executable if present, or None otherwise."""
+    if not cache:
+        return None
```

Comment:
Let's actually change this to `assert cache is not None`, meaning the caller is responsible for calling `initialize_cache` before calling `get_executable` (or not calling `get_executable` at all if they don't wanna initialize the cache). Also note this requirement in the docstring, something like: "`initialize_cache` must be called before calling this function."

Reply:
Is there a way we can do both? I do want to return None if there is not a cache because my logic for the pmap integration handles this but if I just use an assert statement then I  the logic wont go through with the pmap integration it will just print out this message...although maybe that is preferable...I'm not sure, what do you think?